openvpn/src/main/cpp/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ if (NOT ${CMAKE_LIBRARY_OUTPUT_DIRECTORY} MATCHES "build/intermediates/cmake/.*n
     #INCLUDE(${SWIG_USE_FILE})
     #SET_SOURCE_FILES_PROPERTIES(openvpn3/javacli/ovpncli.i PROPERTIES SWIG_FLAGS "-package net.openvpn.ovpn3")
     #SET_SOURCE_FILES_PROPERTIES(openvpn3/javacli/ovpncli.i PROPERTIES CPLUSPLUS ON)
-    #SWIG_ADD_MODULE(ovpen3cli java openvpn3/javacli/ovpncli.i)
+    #SWIG_ADD_MODULE(ovpn3cli java openvpn3/javacli/ovpncli.i)
```

Comment:
This appears to be a third party object. I tried to drop most of the third party code, but it wasn't easy for me to determine which items were...
https://github.com/schwabe/ics-openvpn/commit/f3c03a37a92ca0a69ed9ba49f720a4671899f84a